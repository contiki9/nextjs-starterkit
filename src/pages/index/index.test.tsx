import React from 'react'

import { render } from 'src/__tests__/utils'
import HomePage from 'src/pages/index'

test('ホームページ', () => {
  const { container } = render(<HomePage />)
  expect(container).toMatchSnapshot()
})
