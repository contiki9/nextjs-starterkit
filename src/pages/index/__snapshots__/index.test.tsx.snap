// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ホームページ 1`] = `
<div>
  <div
    class="css-u6v8er"
  >
    <div
      class="css-oadac5"
    >
      contents here
      <button
        class="chakra-button css-yh9qm"
        type="button"
      >
        Button
      </button>
    </div>
  </div>
</div>
`;
